qa/tasks/ceph_manager.py

```diff
@@ -1582,17 +1585,12 @@ def run_ceph_w(self, watch_channel=None, shell=False):
                               'cluster', 'audit', ...
         :type watch_channel: str
         """
-        args = ["sudo",
-                "daemon-helper",
-                "kill",
-                "ceph",
-                '--cluster',
-                self.cluster,
-                "-w"]
+        args = self.run_ceph_w_prefix + ['-w']
         if watch_channel is not None:
             args.append("--watch-channel")
             args.append(watch_channel)
-        return self.controller.run(args=args, wait=False, stdout=StringIO(), stdin=run.PIPE)
+        return self.controller.run(args=args, wait=False, stdout=StringIO(),
+                                   stdin=run.PIPE, shell=shell)
```

Comment:
https://tracker.ceph.com/issues/50178

Looks like this was the actual cause, not #38443 ?

Reply:
> Looks like this was the actual cause, not #38443 ?

Yes. But AFAIS, teuthology's run.run() requires an update. Created a PR - https://github.com/ceph/teuthology/pull/1639.